jsonschema-core.xml

```diff
@@ -1881,17 +1888,24 @@
                 This section defines a vocabulary of applicator keywords that
                 are RECOMMENDED for use as the basis of other vocabularies.
             </t>
+            <t>
+                Meta-schemas that do not use "$vocabulary" SHOULD be considered to
+                require this vocabulary as if its URI were present with a value of true.
+            </t>
             <t>
                 The current URI for this vocabulary, known as the Applicator vocabulary, is:
-                <eref target="https://json-schema.org/draft-08/vocabularies/applicator"/>.
+                <eref target="https://json-schema.org/draft/2019-03/vocab/applicator"/>.
             </t>
             <t>
                 The current URI for the corresponding meta-schema is:
-                <eref target="https://json-schema.org/draft-08/applicator"/>.
+                <eref target="https://json-schema.org/draft/2019-03/meta/applicator"/>.
```

Comment:
Maybe not in this PR, but should we have language identifying this `vocab`/`meta` arrangement as a suggested pattern?

Reply:
Possibly- definitely not in this PR but it's an interesting idea.  Might be better in a best practices section on the site or understanding JSON Schema?  Not sure- feel free to file it as an issue on the spec repo if you want.